content/docs/command-reference/exp/clean.md

```diff
@@ -0,0 +1,33 @@
+# exp clean
+
+Cleanup internal DVC experiment temporary files.
```

Comment:
DVC experiment -> [experiments] (like we do in the other commands)

minor: can we drop internal? just keep temporary - I think it's more or less clear that they are internal then?